test/pp_api_test/test_tokenizer.cc

```diff
@@ -133,7 +133,7 @@ TEST(OrtxTokenizerTest, Phi3_S_Tokenizer) {
   EXPECT_NE(tokenizer, nullptr);
 
   std::vector<extTokenId_t> EXPECTED_IDS_0 = {2028, 374, 264, 1296, 13};
-  std::vector<std::string_view> input = {"This is a test.", "the second one",
+  std::vector<std::string_view> input = {"This is a test.", "Ich liebe München",
```

Comment:
I don't think the problem is related to this regular expression. Is it just an enhancement of unit test?